lib/src/bindings_player_ffi.dart

```diff
@@ -739,6 +739,55 @@ class FlutterSoLoudFfi {
       'setVolume');
   late final _setVolume = _setVolumePtr.asFunction<int Function(int, double)>();
 
+  /// Get a sound's current pan setting.
+  ///
+  /// [handle] the sound handle.
+  /// Returns the range of the pan values is -1 to 1, where -1 is left, 0 is
+  /// middle and and 1 is right.
+  double getPan(int handle) {
+    // Note that because of the float<=>double conversion precision error
+    // (SoLoud lib uses floats), the returned value is not precise. Here we set
+    // a rounding of 5 digits
+    final ret = (_getPan(handle) * 100000).toInt() / 100000;
```

Comment:
Frankly, I don't understand why we need to do this rounding. Is it not enough to just return the double?

Reply:
FWIW, if this is about testing, I'd just document this in the function signature ("Note that if you `setPan()` to `0.8` and then `getPan()`, you might get a slightly different number, such as `0.800000042353`. This is expected since the internal audio engine uses `float` instead of `double`, and so there are rounding errors.").

Then, in unit tests, just use the `closeTo()` operand to deal with the tiny changes.

Reply:
Doing so we add something that the dev should remember when seeing a strange number given back. If you think this is the way, no problem to fix it.

Saying that I realized that this happens also for other functions like `setVolume()` - `getVolume()`! So, that ok :)

Reply:
Yeah, I think it's great to be aware of this and document this, but I don't think it's our place to quantize values for the developer in order to make the values "look right". I totally understand the sentiment, of course. I just think it's adding a functionality people might not necessarily want that needlessly adds more things to the package, too.